docs/pipeline_adms.md

```diff
@@ -0,0 +1,221 @@
+# Pipeline ADMs
+
+Decomposing your ADM into a series of steps (or perhaps even a single
+step) typically will allow you to focus only on the important
+algorithmic code, as we already have several Pipeline ADM steps that
+you can re-use to handle much of the data wrangling that may be
+necessary.  In this document we'll cover what constitutes a pipeline
+ADM, how to create pipeline ADM components (steps), and how to
+configure and run them as an ADM.
+
+**NOTE** - This document assumes you're generally familiar with the
+   system, and how we're using Hydra
+
+## Anatomy of a Pipeline ADM
```

Comment:
The random ADM makes sense as a starting example but I am wondering if the documentation needs a slightly more complex example as well to help demonstrate how to specify aspects like the attribute, structured inference engine, scenario description, prompt, and ICL. I think this readme is pretty clear but if I were referencing it to understand something like the `pipeline_comparative_regression.yaml` I might have remaining questions